lib/tasks/locations.rake

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "nokogiri"
+require "yaml"
+require "alma/config"
+require "httparty"
+
+namespace :locations do
+  desc "Import locations for each library"
+  task :import do
+
+    library_response = HTTParty.get("https://api-na.hosted.exlibrisgroup.com/almaws/v1/conf/libraries?apikey=#{apikey}")
+    library = library_response["libraries"]["library"]
+    library_list = []
+    location_list = {}
+    library.each do |lib|
+      library_list << lib["code"] unless lib["code"] == "EMPTY"
+    end
+
+    library_list.each do |code|
+      location_response = HTTParty.get("https://api-na.hosted.exlibrisgroup.com/almaws/v1/conf/libraries/#{code}/locations?apikey=#{apikey}")
+      locations = location_response["locations"]["location"]
```

Comment:
This is where the `NoMethodError: undefined method `[]' for nil:NilClass` comes from. I used this locally and it worked.
```suggestion
      locations = (location_response.dig("locations","location") || [])
```